server.go

```diff
@@ -561,21 +568,23 @@ func (s *server) Start() error {
 		return err
 	}
 
-	go s.connMgr.Start()
-
 	// If network bootstrapping hasn't been disabled, then we'll configure
 	// the set of active bootstrappers, and launch a dedicated goroutine to
 	// maintain a set of persistent connections.
 	if !cfg.NoNetBootstrap && !(cfg.Bitcoin.SimNet || cfg.Litecoin.SimNet) &&
 		!(cfg.Bitcoin.RegTest || cfg.Litecoin.RegTest) {
 
-		networkBootStrappers, err := initNetworkBootstrappers(s)
+		bootstrappers, err := initNetworkBootstrappers(s)
 		if err != nil {
 			return err
 		}
 
+		// ignore is a set used to keep track of peers already retrieved
+		// from our bootstrappers in order to avoid duplicates.
+		ignore := make(map[autopilot.NodeID]struct{})
```

Comment:
this can be initialized inside `initialPeerBootstrap `?

Reply:
Yep, forgot to move this. Fixed.